search/cypress/support/commands.js

```diff
@@ -74,4 +76,168 @@ Cypress.Commands.add('setAgAndMeta', () => {
     cy.get('.dzu-input').attachFile('test_coco/metadata.json')
     cy.safeSetForm()
     cy.clickText(/^Next$/)
-})
\ No newline at end of file
+})
+
+// CVAT Commands
```

Comment:
Can we import this from the cvat library directly now that it's in a submodule? Copying them here makes them very brittle to a CVAT upgrade.